docs/getting-started/workshops.md

```diff
@@ -1,12 +1,18 @@
 # Workshops
 
-There’s no “right way” to learn how to use the nio Platform. With nearly unlimited applications and wide-ranging skill sets, everyone can and should take a slightly different approach.  
+If you've gotten this far, you should have a [cloud instance](/getting-started/in-the-cloud.html) AND a [local instance](/getting-started/locally.html) up and running... but what are your instances _doing_? Not a lot, yet... but here's where we change that.
```

Comment:
This reads like a guide, docs are not going to be read in order. Instead, this page should describe the workshops and, if necessary, cite the cloud and local instructions as requirements. Is that really a requirement of all workshops though? Or just specific workshops?

Reply:
All workshops require the cloud. Only the distributed demo (plant workshop) requires local install of nio

Reply:
I removed that first paragraph...